packages/backend-modules/access/lib/perks/subscribeToMailJourney.js

```diff
@@ -0,0 +1,39 @@
+const debug = require('debug')('access:lib:perks:subscribeToMailJourney')
+
+const { MAILCHIMP_PROBELESEN_AUDIENCE_ID } = process.env
+
+const give = async (
+  campaign,
+  grant,
+  recipient,
+  settings,
+  t,
+  pgdb,
+  redis,
+  mail,
+) => {
```

Comment:
Same comment as below: it seems a bit wild to me that these arguments are not wrapped in an object like
`({
  campaign,
  grant,
  recipient,
  settings,
  t,
  pgdb,
  redis,
  mail,
}) =>`

BUT I wouldn't change it if this is a common pattern.

Reply:
it's a common pattern in the perks lib